### PR TITLE
ceph-volume: fix fast device alloc size on mulitple device

### PR DESCRIPTION
**Pull request by MrFreezeex**
The size computed by get_physical_fast_allocs() was wrong when the
function had multiple devices to treat.

For instance if there is 4 OSDs and 2 fast devices of each 10G while
allocating 2 slots per fast devvices. The behavior before was that each
slot would be 2.5G meaning that both fast devices would half full. The
behavior now is that each slot will take 5G so that the fast devices
would be full.

Fixes: https://tracker.ceph.com/issues/56031
Signed-off-by: Arthur Outhenin-Chalandre <arthur.outhenin-chalandre@cern.ch>

### CODE REVIEW

**Comment by MrFreezeex** [comment]
jenkins test make check

**Comment by guits** [comment]
jenkins test ceph-volume tox

**Comment by MrFreezeex** [comment]
Ah not sure how I did a typo there between testing locally and pushing the commit :joy: but I fixed it now!

**Comment by MrFreezeex** [comment]
jenkins test api

**Comment by MrFreezeex** [comment]
Hi @guits sorry for the bump but would it be possible for you to review this PR this week since this is quite a regression (at least for us it basically means that we can't expand nor replace any drives without this patch)?

File: src/ceph-volume/tox_install_command.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (guits) [inline src/ceph-volume/tox_install_command.sh:1]: what was the issue with this?

Comment (guits) [reply src/ceph-volume/tox_install_command.sh:1]: yup sounds good to me

**Comment by guits** [comment]
http://pulpito.front.sepia.ceph.com:80/gabrioux-2022-07-18_13:08:51-orch:cephadm-wip-guits-testing-2022-07-18-1119-distro-default-smithi/

**Review by guits — APPROVED** [review]

**Pull request merged**